src/fsharp/FSharp.Build/FSharp.Build.fsproj

```diff
@@ -46,6 +46,7 @@
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" />
     <PackageReference Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
+    <PackageReference Include="System.Collections" Version="4.3.0" />
```

Comment:
Initially I wanted to use a packagereference for System.Collections like other assemblies do, but this didn't work for two reasons: I couldn't use the version from `Versions.props` (it never got picked up?) and the packagereference didn't delegate and add the appropriate gac reference.